src/redux/news/news.actions.js

```diff
@@ -16,29 +18,43 @@ const getNews = () => ({
   type: GET_NEWS
 });
 
-const deleteNewsItem = (payload) => ({
-  type: DELETE_NEWS_ITEM,
+const deleteArticle = (payload) => ({
+  type: DELETE_ARTICLE,
   payload
 });
 
-const addNewsItem = (payload) => ({
-  type: ADD_NEWS_ITEM,
+const addArticle = (payload) => ({
+  type: ADD_ARTICLE,
   payload
 });
 
-const showLoader = () => ({
-  type: SHOW_LOADER
+const setLoading = (loading) => ({
+  type: SET_LOADING,
```

Comment:
It should be called `SET_NEWS_LOADING` to show that it is loading specifically for the news